.gitignore

```diff
@@ -5,6 +5,7 @@ output/
 .idea_modules
 .idea
 .vscode/
+.zed/
```

Comment:
Shouldn't this be in `.git/info/exclude`, since it's user specific and not project specific?